lib/Models/Terria.js

```diff
@@ -262,6 +262,13 @@ var Terria = function(options) {
     this.mapInteractionModeStack = [];
 
     /**
+     * Gets or sets the help screen, a screen which can be used alone or in series to explain a concept in terria in
+     * context.
+     * @type {HelpScreenData[]}
+     */
+    this.helpScreen = [];
```

Comment:
A very minor point, but I think it's good practice to give arrays plural names, eg. `helpScreens` :-)
A related even more minor point - I like to keep the names of the instance the same as the names of the class, but with the first letter lower case - although this is going to get messy with the plural idea - eg. `helpScreenDatas` - or maybe this is really telling us that `HelpScreenData` should be simply `HelpScreen`...